script/Arcueid/Day 12/12_00_ARC12_2.txt

```diff
@@ -731,12 +735,13 @@ The owner of this body, Tohno SHIKI, has vanished just now.
 [sha:b451da08f01d719cb55e3f7fc93289d503ff9d89]{
 -- Page 73, Offset 30741. Glued.
 -- 君への復讐を完遂して、な」
- After carrying out his revenge on you, that is."
+ After carrying out his revenge toward you, that is."
 }
 [sha:beaef76eaa02e186c47a2af45d9bcb5ba203733e]{
 -- Page 74, Offset 30742.
 -- 男は語る。同じ響きでありながら、違う言葉の男の名を。
-The man speaks. With the same resonance, he speaks the name of a different man.
+// 同じ響き here meaning theyre both pronounced shiki? im not too sure how to make it clearer in en text
```

Comment:
I think this is ok, but I think leave the comment because this is kinda important